docs/general/manage-tokens/staking-and-delegating.md

```diff
@@ -1,63 +1,92 @@
+## How to Delegate?
 
-### Tools to use for Staking and Delegation
+Staking can only be performed on the **consensus layer** by the Oasis Wallet -
+Web, Browser extension or the CLI. Some dApps running in ParaTimes may also
+offer staking tokens by implicitly executing the consensus layer staking
+transaction for you.
 
-In addition to using the [Oasis CLI](cli/README.md) for generating the `staking.AddEscrow` transaction, there are a number of wallets and custodians that support staking and delegation on the Oasis Network. Those include:
+You can check out the current validator set, their escrow of staked tokens, the
+commission rate, and the availability in the [Oasis Scan explorer][explorer-validators].
 
-* [Oasis Wallet - Web](oasis-wallets/web.md)
-* [Oasis Wallet - Browser Extension](oasis-wallets/browser-extension.md)
-* [Copper.co](#rewards-and-tokenomics) (custodian)
-* [Anchorage](https://anchorage.com) (custodian)
-* [Finoa](https://finoa.io) (custodian)
-
-You can obtain the list of active validators including their fee and contact information at the Oasis block explorers such as the [Oasis Scan](https://www.oasisscan.com/validators) or the [Oasis Monitor](https://oasismonitor.com/validators). The wallets will also obtain that list from one of the block explorers so you can easily browse through the validators and select the one that you prefer.
+![The validator set in the morning of March 29, 2024](../images/manage-tokens/oasisscan_validators.png)
 
 :::info
 
-Some validators prefer anonymity and they do not list their name or any contact information. In this case only their entity's Oasis address is shown.
+Some validators prefer anonymity and they do not list their name or any contact
+information. In this case only their entity's Oasis address is shown.
 
 :::
 
-You can stop staking and reclaim your tokens at any time. This can be done with a CLI tool or any of the wallets mentioned above. After sending the corresponding `staking.ReclaimEscrow` transaction, your tokens will, to ensure the network security and robustness, enter the **debonding period** defined in the current [genesis](../../node/mainnet/README.md) document. Currently, this period is 336 epochs (around 14 days) and no staking rewards are earned for the duration of this period. Afterwards your ROSE will be free to use.
-
-:::danger
-
-**Staking your ROSE is a different transaction than sending them!** When you stake your tokens (`staking.Escrow` transaction), you can reclaim them at any time. Sending your tokens (`staking.Transfer` transaction) on the other hand means that the **receiver will own the tokens and there is no way of retrieving that tokens back by yourself**.
+Regardless of which validator you pick, **you will earn the same reward as long
+as the validator is online, proposes and signs valid blocks**. We suggest
+that you avoid delegating your tokens to the ones with the largest escrow since
```

Comment:
```suggestion
that you avoid delegating your tokens to the ones with the largest delegations since
```

Reply:
Could we be a little bit stronger and say we recommend that you consider/evaluate validators (without the largest delegations), basically suggest positive priorities.